.github/workflows/push_docs.yaml

```diff
@@ -0,0 +1,126 @@
+name: Push the documentation to Tech-docs
+on:
+  workflow_call:
+    inputs:
+      config:
+        required: true
+        type: string
+      main_branch:
+        default: master
+        required: false
+        type: string
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    env:
+      MAIN_BRANCH: ${{ inputs.main_branch }}
+      TOKEN: ""
+      TECH_DOCS_BRANCH: ""
+      BRANCH_EXISTS: ""
+      CHANGES: ""
+    steps:
+
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - uses: PiwikPRO/actions/github-app-token-generator@master
+        name: Generate token for tech docs pusher
+        id: get-token
+        with:
+          private-key: ${{ secrets.TECH_DOCS_PUSHER_PRIVATE_KEY }}
+          app-id: ${{ secrets.TECH_DOCS_PUSHER_APPLICATION_ID }}
+          repo: PiwikPRO/Tech-docs
+
+      - name: Check out Tech-docs
+        uses: actions/checkout@v3
+        with:
+          repository: 'PiwikPRO/Tech-docs'
+          token: ${{ steps.get-token.outputs.token }}
+          path: 'Tech-docs'
```

Comment:
can we make `Tech-docs` an env? We have a lot of occurrences of this.

Reply:
What about this?

Reply:
I hoped you forgot about this :D

Reply:
As we discussed - not worth it